layouts/index.html

```diff
@@ -11,13 +11,13 @@ <h1 class="font-sans font-body font-normal">
                 {{ .Params.subhead | markdownify }}
             </p>
             <div class="mt-6">
-                <a id="home-hero-cta" href="{{ relref . "/docs/get-started" }}" class="btn bg-purple-300 hover:bg-purple-200 border-none px-8 py-4 transition-all text-sm rounded-full uppercase shadow-lg" title="Create your first project with Pulumi" track-click>
+                <a href="{{ relref . "/docs/get-started" }}" class="btn bg-purple-300 hover:bg-purple-200 border-none px-8 py-4 transition-all text-sm rounded-full uppercase shadow-lg" title="Create your first project with Pulumi">
                     Get Started
                 </a>
             </div>
             <div class="mt-8 text-sm text-purple-100">
-                Pulumi is <a id="home-hero-pricing" class="text-white hover:underline" href="{{ relref . "/pricing#community-edition" }}" track-click>free</a>
-                and <a id="home-hero-github" class="text-white hover:underline" href="https://github.com/pulumi/pulumi" target="_blank" track-click>open source</a>.
+                Pulumi is <a class="text-white hover:underline" href="{{ relref . "/pricing#community-edition" }}">free</a>
```

Comment:
Same question here (why not track), and below, in this file.

Reply:
See above. All links get auto-generated ids. I used specific Ids in cases where I felt it would be needed.